Cargo.toml

```diff
@@ -9,7 +9,7 @@ categories = ["cryptography"]
 description = "Modular password-based encryption"
 
 [dependencies]
-rand_core = "0.4.0"
+rand = "0.7"
```

Comment:
I'd prefer to leave `rand_core` as a crate dependency. If I understand correctly, `rand_core` works better in `no_std` targets (e.g., `wasm-unknown-unknown`), while `rand` can have some problems (at least, that was my experience when tinkering with WASM ~6 months ago; maybe, the situation has improved since then).

On that note, since Rust 1.36 [has stabilized `alloc`](https://blog.rust-lang.org/2019/07/04/Rust-1.36.0.html#the-alloc-crate-is-stable), we can at least think about marking the crate as `no_std`.